tests/test_sensor.py

```
@@ -1,998 +1,936 @@
+    first_started = asyncio.Event()
+    release_first = asyncio.Event()
+    second_ran = asyncio.Event()
+    construct_calls: list[int] = []
 
-    # Stub do_update to avoid executing real logic
-    monkeypatch.setattr(places_instance, "do_update", AsyncMock(return_value=None))
+    class _FakeUpdater:
+        """Update stub that blocks the first call to create a lock queue."""
 
-    background_tasks = set()
+        def __init__(self, **kwargs: object) -> None:
```

Ruff (0.15.20)

[warning] Unused method argument: `kwargs`

(ARG002)